lienzo-core/src/main/java/com/ait/lienzo/client/core/shape/wires/WiresShape.java

```diff
@@ -222,7 +222,7 @@ private void init() {
     private void _loadControls(final IControlHandle.ControlHandleType type) {
         final Map<ControlHandleType, IControlHandleList> handles = getPath().getControlHandles(type);
 
-        if (null != handles) {
+        if (!handles.isEmpty()) {
```

Comment:
if finally staying with the `return null` for the method `getControlHandles` this should be reverted @jomarko , thx!